src/include/reconcile.h

```diff
@@ -31,7 +31,6 @@ struct __wt_reconcile {
     /* Track the page's min/maximum transactions. */
     uint64_t max_txn;
     wt_timestamp_t max_ts;
-    wt_timestamp_t max_ondisk_ts;
```

Comment:
Is this just cleanup? Looks like it's not being read from.

Reply:
Yes, As part of WT-5414 revert, I need to update `max_ondisk_ts` to store durable timestamp instead of commit timestamp.  I feel it is better to delete as it is not used anywhere instead of fixing it.